controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -195,6 +202,9 @@ func (r *DSCInitializationReconciler) authorizationFeatures(instance *dsciv1.DSC
 					servicemesh.EnsureServiceMeshInstalled,
 					servicemesh.EnsureAuthNamespaceExists,
 				).
+				PostConditions(
+					feature.WaitForPodsToBeReady(serviceMeshSpec.ControlPlane.Namespace),
+				).
```

Comment:
It seems it got lost in previous PR

https://github.com/opendatahub-io/opendatahub-operator/blob/odh-2.14.0/controllers/dscinitialization/servicemesh_setup.go#L195C5-L195C74